src/mappings/helpers.ts

```diff
@@ -676,45 +683,58 @@ export function updateDelegationExchangeRate(indexer: Indexer): Indexer {
 }
 
 export function calculatePricePerShare(deployment: SubgraphDeployment): BigDecimal {
+  // Ideally this would be a 1 to 1 recreation of the share sell formula, but due to
```

Comment:
Please add a TODO here with what we talked about over slack. We should come back to this when we have time and figure out why there is a slight deviation